### DISCUSSION TITLE
api 문서는 어디서 볼 수 있나요?

### DISCUSSION THREAD

**Discussion by leewoooo** [naver/ngrinder#841] [Q&A] [answered]
`org.ngrinder.http.HTTPRequest`

위의 package에 해당하는 api를 문서를 볼 수 있는 곳이 있을까요?

**Comment by leewoooo** [comment]
자문 잡답 https://github.com/naver/ngrinder/blob/develop/ngrinder-runtime/src/main/java/org/ngrinder/http/HTTPRequest.java

**Comment by junoyoon** [comment, marked as answer]
https://github.com/naver/ngrinder/blob/develop/ngrinder-runtime/src/main/java/org/ngrinder/http/HTTPRequest.java

**Reply by leewoooo** [reply]
감사합니다 :)
